### DISCUSSION TITLE
✅ Progress Bar on Media Carousel

### DISCUSSION THREAD

**Discussion by jenny-prog** [elementor/elementor#19982] [Feature Request]
Can I know how to add progress bar at bottom of popup video to fast forward the video? Not move to next video but fast forward the video itself. 

![img](https://user-images.githubusercontent.com/61490917/193752446-e2d2a660-6807-48d5-abcf-fef42b6ebd46.jpeg)

Thank you.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new Nested Carousel in Elementor Pro v3.14! 🥳 

You can now select different pagination options in the new Nested Carousel such as Dots, Fraction, and **Progress**.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
